subcommands/waves/init.go

```diff
@@ -50,6 +52,12 @@ func doInitWave(cmd *cobra.Command, args []string) {
 	subcommands.DieNotNil(err, "Version must be an integer")
 	expires := readExpiration(cmd)
 	dryRun, _ := cmd.Flags().GetBool("dry-run")
+	offlineKeysFile, _ := cmd.Flags().GetString("keys")
+	var offlineKeys keys.OfflineCreds
+	if offlineKeysFile != "" {
```

Comment:
how could this be an empty string? If possible, it seems like an error to die on and not a conditional to check?